src/mod_mam_muc_cache_user.erl

```diff
@@ -75,6 +75,7 @@ writer_child_spec() ->
      [?MODULE]}.
 
 start_server(_Host) ->
+    %% XXX make per host server
```

Comment:
What does `XXX` mean here?

Reply:
can be TODO.
XXX signals that it's one of the places, where we don't follow pattern "each host has it's own servers". 
Potentially, can cause issues (cleaning? scalability?).

Reply:
changed :)